core/build.gradle

```diff
@@ -21,6 +21,7 @@ managedDependencies {
         compile "io.netty:$it"
     }
     compile 'io.netty:netty-transport-native-epoll:linux-x86_64'
+    compile 'io.netty:netty-transport-native-kqueue:osx-x86_64'
```

Comment:
Ah guess JAR was the wrong term, meant pom. If this is added as a non-optional dependency in the pom file, then users would have to exclude it manually when creating production jars like a fatjar, and for something like an osx-specific artifact this being the default seems like extra work.